src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -284,11 +290,27 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
             this.props.onSelectedRegionsChanged([]);
         }
 
+        if ((selectedRegions.find((r) => r.category !== selectedRegions[0].category))
```

Comment:
If only region selected and its category is Checkbox, we set tag type to checkbox, otherwise we do nothing